main_logic/tts_client.py

```
@@ -1466,4 +1467,150 @@ async def create_connection():
     try:
         asyncio.run(async_worker())
     except Exception as e:
-        logger.error(f"Local CosyVoice Worker 崩溃: {e}")
\ No newline at end of file
+        logger.error(f"Local CosyVoice Worker 崩溃: {e}")
+        
+def local_qwen3_tts_worker(request_queue, response_queue, audio_api_key, voice_id):
```

Ruff (0.14.14)

[warning] Unused function argument: `voice_id`

(ARG001)